### PR TITLE
Add skeletal setup.py

### PR DESCRIPTION
**Pull request by jakewins**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
```
Comment (nigelsmall) [inline setup.py]: Could you add the standard header containing the license?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "long_description": "", 
+    "author": "Neo Technology",
+    "classifiers": [
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: Apache Software License",
+        "Operating System :: OS Independent",
+        "Topic :: Database",
+        "Topic :: Software Development",
+    ],
+    "zip_safe": False,
```
Comment (nigelsmall) [inline setup.py]: No need to have this. It's zip_safe :-)

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "version": "1.0.0",
+    "description": "Python driver for Neo4j",
+    "long_description": "", 
+    "author": "Neo Technology",
+    "classifiers": [
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: Apache Software License",
+        "Operating System :: OS Independent",
+        "Topic :: Database",
+        "Topic :: Software Development",
```
Comment (nigelsmall) [inline setup.py]: Should add language trove classifiers (https://pypi.python.org/pypi?%3Aaction=list_classifiers)

Aim to support Python 2.6, 2.7, 3.3, 3.4 and 3.5 plus Jython.

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
+
+import os
+import sys
```
Comment (nigelsmall) [inline setup.py]: Neither of these imports are used.

**Comment by spacecowboy** [comment]
Fixed the setup script

**Pull request merged**